tests/test_register.py

```diff
@@ -271,7 +301,75 @@ def test_register_permissions_existing_group_without_intermediate_entries(self):
         path = "/groups/{}/resources/{}/permissions".format(self.test_perm_grp_name, res3_id)
         resp = utils.test_request(self.app, "GET", path)
         body = utils.check_response_basic_info(resp)
-        utils.check_all_equal(body["permission_names"], [res3_perm.value])
+        utils.check_val_is_in(res3_perm, body["permission_names"])
+
+    def test_register_permissions_multiple_resource_type_possible(self):
+        """
+        Verify that service that allows multiple resource-types children retrieves ``type`` field for their creation.
+        """
+        svc_type = ServiceTHREDDS.service_type
+        svc_name = "unittest-service-thredds-register-children-resources"
+        utils.TestSetup.create_TestService(self,
+                                           override_service_name=svc_name,
+                                           override_service_type=svc_type)
+        utils.TestSetup.create_TestGroup(self, override_group_name=self.test_perm_grp_name)
+        session = get_db_session_from_settings(self.app.app.registry.settings)
+
+        res1_name = "test-resource"
+        res2_name = "sub-test-resource"
+        res3_name = "sub-sub-test-resource"
+        res3_perm = "write-deny-recursive"
```

Comment:
Only test write, not read and browse?  Because if one work all the rest guaranty to work?

Reply:
Yes. This test is more to validate the function that parses configs into resource/permission combinations. 
Once this is done, the requests called are the same as all over the other tests cases that validate the various permission combinations.